GitUpKit/Extensions/NSView+Embedding.h

```diff
@@ -0,0 +1,16 @@
+//
+//  NSView+Embedding.h
+//  GitUpKit (OSX)
+//
+//  Created by Dmitry Lobanov on 06.04.2020.
+//
+
+#import <AppKit/AppKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
```

Comment:
i am currently learing gitup so,i like to leave comment on this one ,anyway hello there mate!,.